### PR TITLE
Googledocs 418

### PR DESCRIPTION
**Pull request by DenisGabriela**

### CODE REVIEW

File: Google Docs Share/pom.xml [hunk 1]
```diff
@@ -2,18 +2,19 @@
...
     <modelVersion>4.0.0</modelVersion>
     <artifactId>alfresco-googledocs-share</artifactId>
-    <version>${project.parent.version}-${buildnumber}</version>
+    <version>${project.parent.version}</version>
     <packaging>amp</packaging>
     <name>Alfresco / Google Docs Share Module</name>
 
     <properties>
-        <project.build.finalName>${project.build.finalName}-${buildnumber}.amp</project.build.finalName>
+        <alfresco-community.version>6.0.0-SNAPSHOT</alfresco-community.version>
```
Comment (montgolfiere) [inline Google Docs Share/pom.xml]: I assume there is no Share RC yet ? We will need a TODO reminder (in JIRA) to come back and fix this for 3.1.0-RC2.

File: pom.xml [hunk 2]
```diff
@@ -14,56 +14,20 @@
...
+      <buildnumber>local</buildnumber>
       <alfresco.groupId>org.alfresco</alfresco.groupId>
-      <alfresco-community.version>5.0.c</alfresco-community.version>
-      <alfresco-enterprise.version>5.0</alfresco-enterprise.version>
-      <alfresco.min.version>5.0.0</alfresco.min.version>
+      <alfresco.min.version>6.0.0</alfresco.min.version>
       <alfresco.max.version>6.99.99</alfresco.max.version>
-      <alfresco.client.war.groupId>${alfresco.groupId}</alfresco.client.war.groupId>
-      <alfresco.client.war.version>${alfresco-community.version}</alfresco.client.war.version>
       <alfresco.google.config.version>1.0.0-SNAPSHOT</alfresco.google.config.version>
```
Comment (montgolfiere) [inline pom.xml]: Should be 1.0.0 (not 1.0.0-SNAPSHOT).

File: Google Docs Share/pom.xml (hunk 1 above)
Comment (DenisGabriela) [reply Google Docs Share/pom.xml]: No Share RC is available yet. I've added a TODO.

**Comment by DenisGabriela** [comment]
Thank you for review. I've addressed your comments in the last commit.

**Pull request merged**